pettingzoo/test/api_test.py

```diff
@@ -22,7 +22,29 @@
 list and observation_spaces, action_spaces dictionaries."""
 
 
-def test_observation(observation, observation_0):
+def test_observation(observation, observation_0, env_name=None):
+    env_dicts = [
+        "leduc_holdem_v4",
+        "texas_holdem_no_limit_v6",
+        "texas_holdem_v4",
+        "go_v5",
+        "hanabi_v4",
+        "chess_v5",
+        "connect_four_v3",
+        "tictactoe_v3",
+        "gin_rummy_v4",
+    ]
+    env_graphical_obs = ["knights_archers_zombies_v10"]
+    env_diff_obs_shapes = [
+        "simple_adversary_v2",
+        "simple_world_comm_v2",
+        "simple_tag_v2",
+        "knights_archers_zombies_v10",
+        "simple_push_v2",
+        "simple_speaker_listener_v3",
+        "simple_crypto_v2",
+    ]
+    env_all_zeros_obs = ["knights_archers_zombies_v10"]
```

Comment:
Just a question, is it possible to move all of these definitions outside of the function to the top of the file?